src/rs_quark_assets/components/A2HSBanner.vue

```diff
@@ -0,0 +1,151 @@
+              </div>
+              <div @click="useA2hs()">
+                <p class="text-lg font-medium text-muted">
+                  {{ messages.installPrompt }}
+                </p>
+              </div>
+              <button
+                class="inline-flex"
+                @click="emit('triggered', 'triggered')"
+              >
+                <span class="sr-only">Close</span>
+                <SvgX
+                  class="cursor-pointer text-blue-500 hover:text-blue-700"
+                />
+              </button>
+            </div>
+          </div>
+        </div>
+        <div
+          v-else-if="isIosSafari"
+          aria-live="assertive"
+          class="ios-center max-w-sm overflow-hidden rounded-lg bg-white shadow-lg ring-1 ring-black ring-opacity-5"
+        >
+          <div class="p-4">
+            <div class="flex items-center gap-4">
+              <div class="rounded-full border-2 border-green-400 p-2">
+                <SvgCheck class="cursor-pointer text-green-400" />
+              </div>
+              <div>
+                <p class="text-lg font-medium text-muted">
+                  {{ messages.ios }}
+                </p>
+              </div>
+              <button
+                class="inline-flex"
+                @click="emit('triggered', 'triggered')"
+              >
+                <span class="sr-only">Close</span>
+                <SvgX
+                  class="cursor-pointer text-blue-500 hover:text-blue-700"
+                />
+              </button>
+            </div>
+          </div>
+        </div>
+        <div
+          v-else-if="unsupportedIos"
+          aria-live="assertive"
+          class="max-w-sm overflow-hidden rounded-lg bg-white shadow-lg ring-1 ring-black ring-opacity-5"
+        >
+          <div class="p-4">
+            <div class="flex items-center gap-4">
+              <div class="alert--error rounded-full border-2 p-2">
+                <SvgHandStop class="alert--error cursor-pointer" />
+              </div>
+              <div>
+                <p class="text-lg font-medium text-muted">
+                  {{ messages.iosUnsupported }}
+                </p>
+              </div>
+              <button class="inline-flex" @click="emit('triggered')">
+                <span class="sr-only">Close</span>
+                <SvgX
+                  class="cursor-pointer text-blue-500 hover:text-blue-700"
+                />
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </transition>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "DLA2HS",
+}
+</script>
+
+<script setup>
+import { ref } from "vue"
+
+import SvgCheck from "../assets/images/check.svg"
+import SvgX from "../assets/images/x.svg"
+import Bowser from "bowser"
+
+import SvgHandStop from "../assets/images/hand-stop.svg"
+
+const props = defineProps({
+  installPromptEvent: { type: Object, required: true },
```

Comment:
```suggestion
  promptEvent: { type: Object, required: true },
```

nitpick suggestion: this variable name could be confused with a Function as it can be interpreted as a verb